### DISCUSSION TITLE
Error trying to execute project with the library

### DISCUSSION THREAD

**Discussion by maxi-groldan** [Netflix/dgs-framework#650] [Q&A] [answered]
Hi all.

I am trying to execute a project with com.netflix.graphql.dgs v4.1.0 but I am getting the following error:

Parameter 0 of constructor in com.netflix.graphql.dgs.webmvc.autoconfigure.DgsWebMvcConfigurationProperties required a bean of type 'java.lang.String' that could not be found.

It is a project with Spring Boot v2.1.6.RELEASE
IDE: IntellJ

Does anybody knows what the problem is?

Thanks in advance-

**Comment by paulbakker** [comment, marked as answer]
That's a pretty old Spring Boot version. We currently only test with 2.3 and up. 
I would recommend upgrading Spring Boot.

**Reply by maxi-groldan** [reply]
Hi Paul. I updated Spring Boot to version 2.3.4 and it worked. Many thanks.
